cartoframes/dataset.py

```diff
@@ -7,11 +7,20 @@
 from .columns import normalize_names, normalize_name
 
 from carto.exceptions import CartoException, CartoRateLimitException
+from .geojson import load_geojson
 
 # avoid _lock issue: https://github.com/tqdm/tqdm/issues/457
 tqdm(disable=True, total=0)  # initialise internal lock
 
 
+default_context = None
+
+
+def setDefaultContext(context):
+    global default_context
```

Comment:
❤️ 

Besides, this, it'd be nice to have an option/flag (default to not save) to save it locally for other sessions using the `context.creds.save()` method

Reply:
Something like `save_default_context`?

Reply:
Oh, I see. So with the flag to True the context is also saved. How would be the best way to load this saved context?

Reply:
I'm going to create a ticket to talk about this feature